src/metpy/testing.py

```diff
@@ -21,6 +21,32 @@
 from .units import units
 
 
+def needs_cartopy(test_func):
+    """Mark a test function or fixture as requiring CartoPy.
+
+    If ``cartopy`` is unable to be imported the test, or any test requiring the fixture will
+    be skipped.
```

Comment:
To my eye the docstring here and in `needs_pyproj` below were difficult for me to read correctly on first pass. Might suggest moving the comma or otherwise more clearly separating the clauses of the sentence

> If ``cartopy`` is unable to be imported the test, or any test requiring the fixture, will be skipped.
> If ``cartopy`` is unable to be imported, the test or any test requiring the fixture will be skipped.
> If ``cartopy`` is unable to be imported, the test (or any test requiring the fixture) will be skipped.
> Will skip the test or any test requiring the fixture if ``cartopy`` is unable to be imported.

etc.

Reply:
Tried to address. Let me know how the revised version reads.